From patchwork Wed Aug 15 01:46:40 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Dongli Zhang <dongli.zhang@oracle.com>
X-Patchwork-Id: 10566223
Return-Path: <kvm-owner@kernel.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 40AE71510
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed, 15 Aug 2018 01:47:16 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 2D2192A08D
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed, 15 Aug 2018 01:47:16 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 216272A095; Wed, 15 Aug 2018 01:47:16 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-8.0 required=2.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI,
	UNPARSEABLE_RELAY autolearn=ham version=3.3.1
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id CA9892A08D
	for <patchwork-kvm@patchwork.kernel.org>;
 Wed, 15 Aug 2018 01:47:15 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1728466AbeHOEhE (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Wed, 15 Aug 2018 00:37:04 -0400
Received: from aserp2130.oracle.com ([141.146.126.79]:42360 "EHLO
        aserp2130.oracle.com" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1725831AbeHOEhE (ORCPT <rfc822;kvm@vger.kernel.org>);
        Wed, 15 Aug 2018 00:37:04 -0400
Received: from pps.filterd (aserp2130.oracle.com [127.0.0.1])
        by aserp2130.oracle.com (8.16.0.22/8.16.0.22) with SMTP id
 w7F1i4KC152946;
        Wed, 15 Aug 2018 01:47:03 GMT
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=oracle.com;
 h=from : to : cc :
 subject : date : message-id; s=corp-2018-07-02;
 bh=BF6ckOX6D3HBEMe9A8vgqkrgM2FO83a6w0mt8vEOZ8g=;
 b=jU27pNKUc0S/I5G3N8+blbLCp3XEoQuWfeJV3KVIN97k5nLzHfF4Ws2haT/JgUJRRWnX
 Ly19pki3RsOAQpknhojNA/amizdU6b/xjSFwYjRcEph1bSUzu32Z7hGbsr1rmGF96X2h
 6jw1epvH3dtr9oA9E8lihyJeos9JTFpkR7ckEWIYNp6P0TLNtd5uNuMEEeC6j9dbJ7N4
 Wwkq5teTp2wqoeDDZRN+R4aogdDqoDvyDRxLpiTX+LzzKXKfQhRAy2gve831+5BcdY+Y
 3UVgQD+shjukve0y+X92zEPqglNK60yA2ZWEIBhu0N0G8MRAdJ0wbrhCQJ4zDTbNF269 ag==
Received: from userv0021.oracle.com (userv0021.oracle.com [156.151.31.71])
        by aserp2130.oracle.com with ESMTP id 2ksnad31ff-1
        (version=TLSv1.2 cipher=ECDHE-RSA-AES256-GCM-SHA384 bits=256
 verify=OK);
        Wed, 15 Aug 2018 01:47:03 +0000
Received: from userv0122.oracle.com (userv0122.oracle.com [156.151.31.75])
        by userv0021.oracle.com (8.14.4/8.14.4) with ESMTP id w7F1l2Rn020873
        (version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-GCM-SHA384 bits=256
 verify=OK);
        Wed, 15 Aug 2018 01:47:02 GMT
Received: from abhmp0019.oracle.com (abhmp0019.oracle.com [141.146.116.25])
        by userv0122.oracle.com (8.14.4/8.14.4) with ESMTP id w7F1l2mg031462;
        Wed, 15 Aug 2018 01:47:02 GMT
Received: from linux.cn.oracle.com (/10.182.69.93)
        by default (Oracle Beehive Gateway v4.0)
        with ESMTP ; Tue, 14 Aug 2018 18:47:01 -0700
From: Dongli Zhang <dongli.zhang@oracle.com>
To: kvm@vger.kernel.org, netdev@vger.kernel.org
Cc: linux-kernel@vger.kernel.org, mst@redhat.com, jasowang@redhat.com,
        virtualization@lists.linux-foundation.org
Subject: [PATCH 1/1] vhost: change the signature of __vhost_get_user_slow()
Date: Wed, 15 Aug 2018 09:46:40 +0800
Message-Id: <1534297600-2577-1-git-send-email-dongli.zhang@oracle.com>
X-Mailer: git-send-email 2.7.4
X-Proofpoint-Virus-Version: vendor=nai engine=5900 definitions=8985
 signatures=668707
X-Proofpoint-Spam-Details: rule=notspam policy=default score=0 suspectscore=0
 malwarescore=0
 phishscore=0 bulkscore=0 spamscore=0 mlxscore=0 mlxlogscore=947
 adultscore=0 classifier=spam adjust=0 reason=mlx scancount=1
 engine=8.0.1-1807170000 definitions=main-1808150017
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org
X-Virus-Scanned: ClamAV using ClamSMTP

Remove 'type' from the signature of __vhost_get_user_slow() as it is not
used.

Signed-off-by: Dongli Zhang <dongli.zhang@oracle.com>
---
 drivers/vhost/vhost.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index ed31145..f78d3bc 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -807,8 +807,7 @@ static int vhost_copy_from_user(struct vhost_virtqueue *vq, void *to,
 }
 
 static void __user *__vhost_get_user_slow(struct vhost_virtqueue *vq,
-					  void __user *addr, unsigned int size,
-					  int type)
+					  void __user *addr, unsigned int size)
 {
 	int ret;
 
@@ -846,7 +845,7 @@ static inline void __user *__vhost_get_user(struct vhost_virtqueue *vq,
 	if (uaddr)
 		return uaddr;
 
-	return __vhost_get_user_slow(vq, addr, size, type);
+	return __vhost_get_user_slow(vq, addr, size);
 }
 
 #define vhost_put_user(vq, x, ptr)		\
